spec.html

```diff
@@ -22683,6 +22683,21 @@
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-object.getownpropertydescriptors">
+        <h1>Object.getOwnPropertyDescriptors ( _O_ )</h1>
+        <p>When the `getOwnPropertyDescriptors` function is called, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _obj_ be ? ToObject(_O_).
+          1. Let _ownKeys_ be ? _obj_.[[OwnPropertyKeys]]().
+          1. Let _descriptors_ be ! ObjectCreate(%ObjectPrototype%).
+          1. Repeat, for each element _key_ of _ownKeys_ in List order,
+            1. Let _desc_ be ? _obj_.[[GetOwnProperty]](_key_).
+            1. Let _descriptor_ be ! FromPropertyDescriptor(_desc_).
+            1. Perform ! CreateDataProperty(_descriptors_, _key_, _descriptor_).
```

Comment:
My weak preference is the same as @domenic's - also, this same "issue" already exists with
```js
const obj = {};
Reflect.ownKeys(badProxy).forEach(x => {
  Object.defineProperty(obj, x, Reflect.getOwnPropertyDescriptor(badProxy, x));
});
```